az-digital/az_quickstart
# Closes #1517 Add flexible block migration

## Description


## Related issues




Closes #1517 

## How to test



1. Add diff for this PR as a composer patch to a Quickstart 2 site
2. Try running `migrate:import` on the `az_migration` migration group using a Quickstart 1 source site containing flexible blocks
3. Verify flexible blocks were migrated

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
